samples/highcharts/demo/vertical-sankey/demo.js

```diff
@@ -0,0 +1,300 @@
+// Define a helper to return the formated text
+Highcharts.Templating.helpers.addBR = value => value.split(' ').join('<br/>').toUpperCase();
+
+Highcharts.chart('container', {
+    chart: {
+        type: 'sankey',
+        inverted: true,
+        height: 1000
+    },
+    title: {
+        text: 'Evaluating the energy consumed for water use in the United States in 2010'
```

Comment:
Can we find some recent data? Using data from 2010 sends the unconscious message that we created this demo over a decade ago. Alternatively just remove the year from the title.

Reply:
Sadly, there is no newer data, so I think removing the year would be good for that case.